frontend/src/theme/darkTheme/themeColors.ts

```diff
@@ -26,7 +26,7 @@ const genericColors = {
   info: "#1e6777",
   success: "#176c36",
   warning: "#947c2d",
-  primary: "#f63366",
+  primary: "orange",
```

Comment:
Just a placeholder for now to differentiate. Need to figure out all the colors afterwards